save-backend/src/test/kotlin/org/cqfn/save/backend/controller/CloningRepositoryControllerTest.kt

```diff
@@ -66,23 +63,9 @@ import kotlin.io.path.createFile
 @EnableConfigurationProperties(ConfigProperties::class)
 @TestInstance(TestInstance.Lifecycle.PER_CLASS)
 @MockBeans(
-    MockBean(AgentStatusRepository::class),
-    MockBean(AgentRepository::class),
-    MockBean(ExecutionRepository::class),
     MockBean(ExecutionService::class),
-    MockBean(TestExecutionRepository::class),
-    MockBean(TestRepository::class),
-    MockBean(TestSuiteRepository::class),
-    MockBean(ProjectRepository::class),
-    MockBean(GitRepository::class),
-    MockBean(StandardSuitesUpdateScheduler::class),
-    MockBean(UserRepository::class),
-    MockBean(AwesomeBenchmarksRepository::class),
-    MockBean(OrganizationRepository::class),
-    MockBean(OrganizationController::class),
-    MockBean(OrganizationService::class),
-    MockBean(LnkUserProjectRepository::class),
     MockBean(ProjectPermissionEvaluator::class),
+    MockBean(UserRepository::class),
```

Comment:
Why this repo should be mocked, while other not? What's the difference?

Reply:
Because this one is among dependencies of the controller under test. Other were mocked, because we accidentally enabled auto-configuration for all repositories.